tests/integration/test_speck.py

```diff
@@ -18,6 +20,9 @@
 
 _COMPONENT_ID = 'test-speck'
 
+if not sys.platform.startswith("darwin"):
+    pytest.skip("Skipping broken Speck tests", allow_module_level=True)
```

Comment:
What broke? I'm hesitant to just disable them, we'll never come back to fix them then.